.storybook/themes/news.scss

```diff
@@ -2626,6 +2636,10 @@
 					components: (
 						link: (
 							font-size: var(--body-font-size),
+							filter: drop-shadow(0px 0px 0px rgba(0, 0, 0, 0)),
+						),
+						link-hover: (
+							color: var(--global-neutral-8),
```

Comment:
We don't have to override the default styles here since they are the same

Reply:
Made the changes, looks like it's working fine on my end. Let me know if you have any issues